### PR TITLE
Gearman role

### PR DESCRIPTION
**Pull request by kucerakk**

### CODE REVIEW

File: roles/gearman/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (gtema) [inline roles/gearman/.travis.yml]: please drop this file completely

File: roles/gearman/meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+galaxy_info:
```
Comment (gtema) [inline roles/gearman/meta/main.yml]: this file does not make sense under the collection either

File: roles/gearman/.travis.yml (hunk 1 above)
Comment (kucerakk) [reply roles/gearman/.travis.yml]: file dropped

File: roles/gearman/meta/main.yml (hunk 2 above)
Comment (kucerakk) [reply roles/gearman/meta/main.yml]: file dropped

**Review by gtema — APPROVED** [review]

**Comment by gtema** [comment]
regate

**Comment by gtema** [comment]
regate

**Comment by gtema** [comment]
regate

**Pull request merged**